eng/pipelines/maui-release.yml

```diff
@@ -26,6 +26,7 @@ schedules:
   branches:
     include:
     - main
+    - net9.0
```

Comment:
This won't work with the nightly push job as configured, as it currently relies on artifacts that have been replaced by the drop service in net9.0.

Reply:
oh yeah gotcha , we can have an updated version on net9 right?